<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>622</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Button_load">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Images</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pTextEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_mouse">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>540</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="ImageView" name="particle_im">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_confirm">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>540</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_erase">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>570</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pTEdit_points">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pTEdit_circleCenter">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>240</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_showz">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>show_z</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
